app/controllers/course/admin/admin_controller.rb

```diff
@@ -11,6 +11,15 @@ def update #:nodoc:
     end
   end
 
+  def destroy #:nodoc:
```

Comment:
Same as hound - any ideas on how to simplify? Only one I can think of is to remove the `title` attribute. Not sure if that works >_<